src/Containers.jsx

```diff
@@ -397,7 +398,10 @@ class Containers extends React.Component {
         }
         const info_block = (
             <div className="container-block">
-                <span className="container-name">{container.Name}</span>
+                <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                    <span className="container-name">{container.Name}</span>
+                    {isToolboxContainer && <Badge>toolbox</Badge>}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1409-20230914-111755-35d42e9d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)